packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -479,11 +483,53 @@ export class LoaderContainerTracker implements IOpProcessingController {
 	private async processQueue<U>(
 		containers: IContainer[],
 		getQueue: (container: IContainer) => IDeltaQueue<U>,
+		waitForConnectedContainers = false,
 	) {
 		await this.pauseProcessing(...containers);
 		const resumed: IDeltaQueue<U>[] = [];
 
 		const containersToApply = this.getContainers(containers);
+
+		if (waitForConnectedContainers) {
+			for (const container of containersToApply) {
+				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+				const index = this.containers.get(container)!.index;
+
+				// ** Start back compat
+				// Before 0.54.x, the runtime doesn't proactive in disconnecting and reconnecting when there are ops
```

Comment:
```suggestion
				// Before 0.54.x, the runtime isn't proactive in disconnecting and reconnecting when there are ops
```

Reply:
Sorry... I have change the implementation a little bit and this doesn't exist anymore.